Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM debian:stable-slim as build
 WORKDIR /usr/src
 ENV DEBIAN_FRONTEND=noninteractive
 RUN apt update -y
-RUN apt install -y curl cmake libgl1-mesa-dev libsdl2-dev libcurl4-openssl-dev libopenal-dev libfreetype6-dev mingw-w64 g++-mingw-w64 g++-multilib git zip vim-nox rsync
+RUN apt install -y make gcc g++ libgl1-mesa-dev libsdl2-dev libcurl4-openssl-dev libopenal-dev libfreetype6-dev zip rsync
```

Comment:
Can you make a new layer for the Tremded and Tremulous binaries which removes build tools from the final image.